### DISCUSSION TITLE
Workflow Opensource

### DISCUSSION THREAD

**Discussion by Maskowh** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#111] [General]
Bonjour,

Je me demandais si vous aviez un workflow défini sur l'Opensource.

J'ai fait quelques premières contribs en faisant directement le Dev et la PR, car les devs étaient très petits et me servaient à illustrer mes propositions. (sans validation préalable du staff du coup)

Jusqu'à cette PR #90 qui n'a pas convaincu le staff. J'ai fait une nouvelle [proposition](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/pull/90#issuecomment-1416274152), mais comme c'est déjà dans la PR, je ne sais pas si c'est passé à la trappe

Je me suis donc dit que, pour mes prochains devs, j'allais créer les issues et attendre validation avant de me lancer dans les devs, passant sur un workflow du genre: Proposer un Improvement/Feature en Issue -> Refusé/discuté/approuvé par Staff/Admin -> Développé -> PR -> Review par les devs.

Mais je ne sais pas si c'est ce que vous attendez de votre coté, ou si vous attendez que le dev soit fait avant de regarder.

J'ai bien conscience que vous devez être focus sur le DLC2 et donc que n'avez pas forcément le temps de regarder les issues qui ne sont pas encore dev, mais j'aimerais juste m'assurer d'être sur la même longueur d'onde que vous:
Est ce que je dois dev mes propositions avant que vous ne les regardiez?
ou
Est-ce que vous regardez quand-même les issues dont les devs ne sont pas encore réalisés pour approuver/refuser les propositions avant les devs?

Je suis désolé si je paraît "impatient" avec ce message, ce n'est pas du tout le cas, j'ai bien conscience que la priorité c'est le DLC2 et pas des p'tites améliorations téléphone. Je veux juste savoir si vous avez un workflow précis à respecter, si les Issues que j'ai créé en attente sont "suffisantes" pour que vous les regardiez.

Merci!

**Comment by NariieL** [comment]
Hello !

`Est-ce que vous regardez quand-même les issues dont les devs ne sont pas encore réalisés pour approuver/refuser les propositions avant les devs?`

Ceci est la meilleure des choses à faire. Il est inutile que tu te lances dans du dev sans qu'on ait donné l'accord, car tu risques de mettre à la poubelle du temps perdu, que tu aurais pu investir ailleurs, même pas forcément sur SOZ. 

On ne souhaite pas que vous perdiez du temps inutilement à cause de "nous et nos décisions", donc tu fais bien de toujours créer une issue et d'y attendre une réponse avant de te lancer sur le développement. 🙏🏼

Pour le reste, tu as parfaitement résumé, tout notre focus est sur le DLC 2. Merci de ta patience !

**Reply by Maskowh** [reply]
Merci pour la réponde, bon courage à vous!
